components/bulletins/BulletinCard.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div v-show="!!title" :class="classObject">
+        <h2>
+            {{ $t(title) }}
+        </h2>
+        <p>{{ $t(description) }}</p>
+        <a :href="linkUrl" :title="linkStr">{{ linkStr }}</a>
+        <!-- <div class="text-center">
+            <text-button :to="linkUrl">{{ linkStr }}</text-button>
+        </div> -->
```

Comment:
1. Why do we need to specify `:title="linkStr"` here?
2. According to 1., instead of calling it `linkStr`, could it be called `linkTitle`? Many things could be `XXStr`. Giving it the meaning or the purpose is a better variable naming strategy.
3. Could we use the `text-button` component instead of re-styling the anchor over again? Otherwise, it'll be hard to maintain. Also, the style you added is different from what the spec needs.
4. Even though the `text-button` component style isn't 100% the same as the design spec, but there's already a `secondary` variant for you to utilize. No extra styling is needed.
5. I'm guessing the reason why you gave up on using the `text-button` component is that the links could be external or internal. Regardless, this is an issue of the way you constructed the `.i18n.js`. Consider refactoring the structure using `linkTitle`, `linkHref`, `linkTo`. Please talk to me if you can't reckon this approach.